PoshSSDTBuildDeploy/Functions/PublishDatabaseDeployment.ps1

```diff
@@ -14,46 +14,31 @@ Function Publish-DatabaseDeployment {
 
 
     )
-    if ($ScriptPath) {
-        if (-not (Test-Path $ScriptPath)) {
-            Throw "Script Path Invalid!"            
-        }
-    }
+    
     Write-Verbose 'Testing if DACfx was installed...'
-    Write-Verbose $dacfxPath
-    if (!$dacfxPath) {
-        throw 'No usable version of Dac Fx found.'
-    }
-    else {
-        try {
-            Write-Verbose 'DacFX found, attempting to load DAC assembly...'
-            Add-Type -Path $dacfxPath
-            Write-Verbose 'Loaded DAC assembly.'
-        }
-        catch [System.Management.Automation.RuntimeException] {
-            throw "Exception caught: " + $_.Exception.GetType().FullName
-        }
-    }
-    if (Test-Path $dacpac) {
-        $dacPackage = [Microsoft.SqlServer.Dac.DacPackage]::Load($Dacpac)
-        Write-Host ('Loaded dacpac ''{0}''.' -f $Dacpac) -ForegroundColor White -BackgroundColor DarkMagenta
-    }
-    else {
-        Write-Verbose "$dacpac not found!" -Verbose
-        throw
-    }
-    if (Test-Path $publishXml) {
-        $dacProfile = [Microsoft.SqlServer.Dac.DacProfile]::Load($publishXml)
-        Write-Host ('Loaded publish profile ''{0}''.' -f $publishXml) -ForegroundColor White -BackgroundColor DarkMagenta
+    if (-not (Test-Path $dacfxPath)) { throw "No usable version of Dac Fx found at $dacfxPath" }
+    if (-not (Test-Path $dacpac)) { throw "$dacpac not found!" }
+    if (-not (Test-Path $publishXml)) { throw "$publishXml not found!" }
+    if (-not (Test-Path $ScriptPath)) { Throw "Script Path Invalid!" }
+
+    try {
+        Write-Verbose 'DacFX found, attempting to load DAC assembly...'
+        Add-Type -Path $dacfxPath
+        Write-Verbose 'Loaded DAC assembly.'
     }
-    else {
-        Write-Verbose "$publishXml not found!" -Verbose
-        throw
+    catch [System.Management.Automation.RuntimeException] {
+        throw ("Exception caught: {0}" -f $_.Exception.GetType().FullName)
     }
+    
+    $dacPackage = [Microsoft.SqlServer.Dac.DacPackage]::Load($Dacpac)
+    Write-Host ("Loaded dacpac '{0}'." -f $Dacpac) -ForegroundColor White -BackgroundColor DarkMagenta
+    
+    $dacProfile = [Microsoft.SqlServer.Dac.DacProfile]::Load($publishXml)
+    Write-Host ("Loaded publish profile '{0}'." -f $publishXml) -ForegroundColor White -BackgroundColor DarkMagenta
     if ($getSqlCmdVars) {
-        Get-SqlCmdVars $dacProfile.DeployOptions.SqlCommandVariableValues
+        Get-SqlCmdVars $dacProfile.DeployOptions.SqlCommandVariableValues    
     }
-    $timeStamp = (Get-Date).ToString("yyMMdd_HHmmss_f")    
+    $timeStamp = Get-Date -Format "yyMMdd_HHmmss_f"
```

Comment:
The previous way is more a C# way I just changed it to be consistent with powershell